dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HeartBeat.java

```diff
@@ -245,18 +245,18 @@ public static HeartBeat decodeHeartBeat(String heartBeatInfo) {
         HeartBeat heartBeat = new HeartBeat();
         heartBeat.cpuUsage = Double.parseDouble(parts[0]);
         heartBeat.memoryUsage = Double.parseDouble(parts[1]);
-        heartBeat.diskAvailable = Double.parseDouble(parts[2]);
-        heartBeat.loadAverage = Double.parseDouble(parts[3]);
-        heartBeat.availablePhysicalMemorySize = Double.parseDouble(parts[4]);
-        heartBeat.maxCpuloadAvg = Double.parseDouble(parts[5]);
-        heartBeat.reservedMemory = Double.parseDouble(parts[6]);
-        heartBeat.startupTime = Long.parseLong(parts[7]);
-        heartBeat.reportTime = Long.parseLong(parts[8]);
-        heartBeat.serverStatus = Integer.parseInt(parts[9]);
-        heartBeat.processId = Integer.parseInt(parts[10]);
-        heartBeat.workerHostWeight = Integer.parseInt(parts[11]);
-        heartBeat.workerExecThreadCount = Integer.parseInt(parts[12]);
-        heartBeat.workerWaitingTaskCount = Integer.parseInt(parts[13]);
+        heartBeat.loadAverage = Double.parseDouble(parts[2]);
+        heartBeat.availablePhysicalMemorySize = Double.parseDouble(parts[3]);
+        heartBeat.maxCpuloadAvg = Double.parseDouble(parts[4]);
+        heartBeat.reservedMemory = Double.parseDouble(parts[5]);
+        heartBeat.startupTime = Long.parseLong(parts[6]);
+        heartBeat.reportTime = Long.parseLong(parts[7]);
+        heartBeat.serverStatus = Integer.parseInt(parts[8]);
+        heartBeat.processId = Integer.parseInt(parts[9]);
+        heartBeat.workerHostWeight = Integer.parseInt(parts[10]);
+        heartBeat.workerExecThreadCount = Integer.parseInt(parts[11]);
+        heartBeat.workerWaitingTaskCount = Integer.parseInt(parts[12]);
+        heartBeat.diskAvailable = Double.parseDouble(parts[13]);
```

Comment:
You just change the position and do nothing on it? Am I right?

Reply:
Yes. I didn't think there was a quote anywhere else.
Switch to version 2.0.6 if you can

Reply:
do you mean you want to add disk caption check but when you submit PR and find it already in `dev` branch?

Reply:
What I added on the 2.0.6 branch was merged, and then I submitted it to version 3.0 and found a bug! Now I want to fix it

Reply:
In this case you should change your target branch to `dev`

Reply:
I wrote this function on branch 2.0.6, and it also fits this branch!  I have also committed the ** edge of dev, but it is not merged yet